ssv/partial_sig_container.go

```diff
@@ -43,10 +29,43 @@ func (ps *PartialSigContainer) ReconstructSignature(root [32]byte, validatorPubK
 	return signature.Serialize(), nil
 }
 
-func (ps *PartialSigContainer) HasQuorum(root [32]byte) bool {
-	return uint64(len(ps.Signatures[rootHex(root)])) >= ps.Quorum
+// SignatureForRoot returns a map of signer and signature for a specific root
+func (ps PartialSignatureContainer) SignatureForRoot(root [32]byte) map[types.OperatorID][]byte {
```

Comment:
not trying to nit pick but I think this function should be named 
```suggestion
func (ps PartialSignatureContainer) SignaturesForRoot(root [32]byte) map[types.OperatorID][]byte {
```

Reply:
done